integrations/opennms-vmware/src/main/java/org/opennms/netmgt/provision/service/vmware/VmwareImporter.java

```diff
@@ -249,6 +250,11 @@ private RequisitionNode createRequisitionNode(Set<String> ipAddresses, ManagedEn
         // Foreign Id consisting of managed entity Id
         requisitionNode.setForeignId(managedEntity.getMOR().getVal());
 
+        // set the location
+        if (!Strings.isNullOrEmpty(request.getLocation())) {
```

Comment:
FYI we have a `StringUtils` in core.lib that can do this without having to pull in Guava... (yes, I'm still on about that, lol)